src/test/resources/application.yaml

```diff
@@ -0,0 +1,186 @@
+    #    graphql_enabled: true
+    #    narrative_enabled: true
+    #    mdm_enabled: true
+    #    local_base_urls:
+    #      - https://hapi.fhir.org/baseR4
+    mdm_enabled: false
+    #    partitioning:
+    #      allow_references_across_partitions: false
+    #      partitioning_include_in_search_hashes: false
+    #cors:
+    #  allow_Credentials: true
+      # These are allowed_origin patterns, see: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/cors/CorsConfiguration.html#setAllowedOriginPatterns-java.util.List-
+    #  allowed_origin:
+    #    - '*'
+
+    # Search coordinator thread pool sizes
+    search-coord-core-pool-size: 20
+    search-coord-max-pool-size: 100
+    search-coord-queue-capacity: 200
+
+    # Threadpool size for BATCH'ed GETs in a bundle.
+    #    bundle_batch_pool_size: 10
+    #    bundle_batch_pool_max_size: 50
+
+    #    logger:
+    #      error_format: 'ERROR - ${requestVerb} ${requestUrl}'
+    #      format: >-
+    #        Path[${servletPath}] Source[${requestHeader.x-forwarded-for}]
+    #        Operation[${operationType} ${operationName} ${idOrResourceName}]
+    #        UA[${requestHeader.user-agent}] Params[${requestParameters}]
+    #        ResponseEncoding[${responseEncodingNoDefault}]
+    #      log_exceptions: true
+    #      name: fhirtest.access
+    #    max_binary_size: 104857600
+    #    max_page_size: 200
+    #    retain_cached_searches_mins: 60
+    #    reuse_cached_search_results_millis: 60000
+    tester:
+      home:
+        name: Local Tester
+        server_address: 'http://localhost:8080/fhir'
+        refuse_to_fetch_third_party_urls: false
+        fhir_version: R4
+      global:
+        name: Global Tester
+        server_address: "http://hapi.fhir.org/baseR4"
+        refuse_to_fetch_third_party_urls: false
+        fhir_version: R4
+#    validation:
+#      requests_enabled: true
+#      responses_enabled: true
+#    binary_storage_enabled: true
+#    bulk_export_enabled: true
+#    subscription:
+#      resthook_enabled: true
+#      websocket_enabled: false
+#      email:
+#        from: some@test.com
+#        host: google.com
+#        port:
+#        username:
+#        password:
+#        auth:
+#        startTlsEnable:
+#        startTlsRequired:
+#        quitWait:
+#    lastn_enabled: true
+#    store_resource_in_lucene_index_enabled: true
+###  This is configuration for normalized quantity serach level default is 0
+###   0: NORMALIZED_QUANTITY_SEARCH_NOT_SUPPORTED - default
+###   1: NORMALIZED_QUANTITY_STORAGE_SUPPORTED
+###   2: NORMALIZED_QUANTITY_SEARCH_SUPPORTED
+#    normalized_quantity_search_level: 2
+#elasticsearch:
+#  debug:
+#    pretty_print_json_log: false
+#    refresh_after_write: false
+#  enabled: false
+#  password: SomePassword
+#  required_index_status: YELLOW
+#  rest_url: 'localhost:9200'
+#  protocol: 'http'
+#  schema_management_strategy: CREATE
+#  username: SomeUsername
```

Comment:
I think we should have this converted to regular spring boot ElasticsearchRestClientAutoConfiguration but that is for another time to do so